tuna/miopen/miopen_tables.py

```diff
@@ -355,22 +357,25 @@ def session(self):
     """session key"""
     return Column(Integer, ForeignKey("session.id"), nullable=False)
 
+  reason = Column(String(length=60), nullable=False, server_default="")
   state = Column(Enum(JobEnum), nullable=False, server_default="new")
+  retries = Column(Integer, nullable=False, server_default="0")
+  result = Column(Text, nullable=True)
```

Comment:
needs and alembic change, here and other cols you added/removed bellow

Reply:
How to run alembic?

Reply:
Columns were moved in code, but no additions / deletions.